sdk/lusid/api_client.py

```diff
@@ -326,6 +363,10 @@ def call_api(self, resource_path, method,
                                  number provided, it will be total request
                                  timeout. It can also be a pair (tuple) of
                                  (connection, read) timeouts.
+        :param _request_auth: set to override the auth_settings for an a single
+                              request; this effectively ignores the authentication
+                              in the spec for a single request.
```

Comment:
Okay looks like this is what that new `_request_auth` parameter is, it allows you to override the authentication to use for a single API request. You must provide it on an api call as a keyword argument of `_request_auth=`